recipes/xsd/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "xsd" %}
+{% set version = "4.0.0" %}
+{% set sha256 = "eca52a9c8f52cdbe2ae4e364e4a909503493a0d51ea388fc6c9734565a859817" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.codesynthesis.com/download/{{ name }}/4.0/{{ name }}-{{ version }}+dep.tar.bz2
+  sha256: {{ sha256 }}
+
+  patches:
+    - osx-build.patch
```

Comment:
It's from the mailing list, too. i added a comment and also referred to the homebrew formula the thread in the mailinglist was about.